.circleci/config.yml

```diff
@@ -370,4 +383,7 @@ workflows:
           requires:
             - test_end_to_end
             - deploy_server
-          <<: *deploy_filter
\ No newline at end of file
+          <<: *deploy_filter
+      - lighthouse:
+          requires:
+            - deploy_client
```

Comment:
Oh wait, one more thought, would it be fine to only run this job on the master branch? It's still 10+ minutes, right? Don't want it clogging up CI.